libbeat/monitoring/monitoring.go

```diff
@@ -4,6 +4,7 @@ import "errors"
 
 type Mode uint8
 
+//go:generate stringer -type=Mode
```

Comment:
yeah, I wouldn't add it to `make update`. I have had the build broken when just doing unit tests without having build the full project before `go generate`. Maybe introduce separate `make go-generate`, but this one might fail and developer has to 'fix' it locally. generated files should be committed.